README.md

```diff
@@ -69,6 +71,10 @@ After writing this method, our new result should be:
 
 **think**: Where should we call `strip_punctuation`? Does it matter? Why?
 
+I chose to call it in `#print_all_moods` because it only needs to modify the local copy that the method uses. However, since I want `#print_all_moods` to do something with the modified data, I put it outside the loop that would analyze the strings. 
+
+This usage also seems in keeping with the idea that we should do our best to call methods and not data, as per our discussions about the Metz book. (Rather than trusting a second variable is reliable, and storing the result of `#strip_punctuation` in it, I am calling the method directly on the data to be analyzed. I hope I'm expressing myself.)
```

Comment:
I'm not totally following your explanation, but you put it in the place I wanted... I think for the reasons I wanted.